src/modules/common/styles/utilities/_global-vars.scss

```diff
@@ -1,3 +1,3 @@
 /* Global variables go here */
 
-$background-color: #FAFAFA;
\ No newline at end of file
+$background-color: #FAFAFA;
```

Comment:
May be simpler just to dump this file and keep everything in base.scss since I doubt we'll be doing a crazy amount of global vars/styles.